### ISSUE TITLE
Captcha.

### ISSUE THREAD

**Issue by Overlordmittenz** [odsum/TinyChatBot#13]
Captcha is nearly impossible to bypass. Possible to have the script open the browser in order to solve the captcha, removing the long process of trying to bypass the captcha?

**Comment by sean-james** [comment]
Unfortunately not, Tinychat now uses the new captcha identification algorithms and they are kept a secret so all we can really do is make guesses about how it works. Thus so far no workaround at this point.  My understanding is that the new system will analyze your activity across all of Google's services.

Thus, it is likely that the checks are not limited to just the page that has the checkbox on it. For example, if they detect that your computer/IP address you are using was also used in the past to do things that a normal human would do - things like checking Gmail, searching on Google search, uploading files to Drive, sharing photos, browsing the web etc. - then it can probably be reasonably sure that you are a human and allow you to skip the image verification. On the other hand, if it can't associate your computer with any previous human-like activity, then it would be more suspicious and give you the image verification. Though the mouse behaviour as it clicks the checkbox may be one factor it analyzes, there is almost certainly a lot more to it.

In short, if you're looking to host it via a hosting provider you will need to verify the captcha form from where you're hosting the bot from somehow. If you're hosting it from your computer, however, that should be fine. 

Good luck!

**Comment by Overlordmittenz** [comment]
Developed a workaround for this and got it hosted on a vps. I appreciate the feedback!

**Comment by spiderfaced** [comment]
Can you give some advice on how I might going about doing this? I'm fine with doing it locally and opening a browser every time I open the bot like you mentioned.
